eclipse/tern.eclipse.ide.server.nodejs.core/src/tern/eclipse/ide/server/nodejs/core/NodejsDebuggersManager.java

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (c) 2015 Genuitec LLC.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Piotr Tomiak <piotr@genuitec.com> - initial API and implementation
+ */
+package tern.eclipse.ide.server.nodejs.core;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import tern.eclipse.ide.server.nodejs.internal.core.debugging.webclipse.WebclipseDebugger;
+
+public class NodejsDebuggersManager {
+
+	private static Map<String, INodejsDebugger> debuggers = new LinkedHashMap<String, INodejsDebugger>();
+
+	static {
+		// Add supported debuggers
+		addDebugger(new WebclipseDebugger());
```

Comment:
I wanted to avoid any dependencies on debugger plugins, so that they do not have to be modified in any way, or even depend on tern.java during compilation time. I don't like hardcoded things, but in this case an extension point consumed solely by the plugin providing it would be an overkill. However, if there will be a need for other debuggers to use an extension point, and depend on Tern Node.js plugin, one who develops it, can always enhance NodejsDebuggersManager.